.pylintrc

```diff
@@ -41,7 +41,7 @@ load-plugins=linter_plugin
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used,abstract-class-little-used,bad-continuation,too-few-public-methods,no-self-use,invalid-name,too-many-instance-attributes,cyclic-import
+disable=fixme,locally-disabled,abstract-class-not-used,abstract-class-little-used,bad-continuation,too-few-public-methods,no-self-use,invalid-name,too-many-instance-attributes,cyclic-import,duplicate-code
```

Comment:
The reason was setUp methods in tests, mocking the environment through distribution fingerprinting and mock constants. I moved the mocks to `get_apache_configurator` in tests.util.
This also helped us get rid of a lot of test setup boilerplate! 

Other part was doing the the same testcase in all of the override tests where we check that all the defined vhosts are found in the configuration. I just renamed variables in these tests, as I think it's better to keep the tests in the actual test files instead of running test from external file.